src/main/java/newbank/server/commands/HelpCommand.java

```diff
@@ -0,0 +1,51 @@
+package newbank.server.commands;
+
+import newbank.server.CustomerID;
+import newbank.server.NewBank;
+
+public class HelpCommand extends Command {
+  public HelpCommand(final NewBank bank, final String[] tokens, final CustomerID customer) {
+  }
+
+  @Override
+  public String execute() {
```

Comment:
Rather than displaying the syntax name, which they can get by issuing "COMMAND HELP", we could display,
"COMMAND  a brief explanation of the command".
Try "git help" on the command line and the output for inspiration.

Regards,
Jules